calendar_v17/odoo_office365_calendar/static/description/index.html

```
@@ -0,0 +1,785 @@
+                                                                        style="font-weight: bold;font-family:'Montserrat', sans-serif">
+                                                                        <a style="color:#000;"
+                                                                           href="https://apps.odoo.com/apps/modules/17.0/web_auto_reload/"
+                                                                           target="_blank"
+                                                                           style="font-family:'Montserrat', sans-serif;">
+                                                                            <img class="img m-auto shadow"
+                                                                                 style="width:120px;font-size:16px"
+                                                                                 src="images/web_auto_icon.png"
+                                                                                 alt="Web Auto Refresh and Reload"/>
+                                                                            <div class="text-center mt-4"
+                                                                                 style="font-size:16px;">
+                                                                                <b>Web Auto<br>Refresh</br> and Reload</b>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </br> ]

(tag-pair)